ChameleonForms/Component/Config/FieldConfiguration.cs

```diff
@@ -379,6 +379,18 @@ public interface IFieldConfiguration : IHtmlString
         /// </summary>
         /// <returns>A readonly field configuration</returns>
         IReadonlyFieldConfiguration ToReadonly();
+
+        /// <summary>
+        /// Excludes one or more Enum values from the generated field.
+        /// </summary>
+        /// <param name="enumValues">The value of Enum(s) to exclude from the generated field.</param>
+        /// <returns></returns>
+        IFieldConfiguration ExcludeValues(params Enum[] enumValues);

```

Comment:
Pity we can't have type inference.

Do you think ExcludeValues is discoverable enough?